config/constants.php

```diff
@@ -376,4 +376,9 @@
       'login' => '/assets/images/auth/Login_bg.jpg',
       'inactive' => '/assets/images/auth/Login_bg.jpg',
     ],
+    'aops' => [
+        'BC' => 'Blaine County',
+        'LS' => 'Los Santos',
+        'SW' => 'State Wide',
```

Comment:
I don't see a point for there to be a State-Wide AOP selection since we already have the ability to choose both Los Santos AND Blaine County.